package/MDAnalysis/lib/util.py

```diff
@@ -203,6 +204,8 @@
 from functools import wraps
 import textwrap
 import weakref
+import copy
+import shutil
```

Comment:
Why is `shutil` added if it's not used in your code?

Reply:
oops here and below are leftover imports from me testing out various things - should be fixed now.